resources/install.rb

```diff
@@ -0,0 +1,53 @@
+# resources/install.rb
+
+# Resource for InfluxDB install
+
+property :influxdb_key, String, default: 'https://repos.influxdata.com/influxdb.key'
+property :arch_type, String, default: 'amd64'
+default_action :install
+
```

Comment:
I can certainly do that I didn't because there was no concept in the cookbook of installing not from a repo so there was no reason to add it but I can add some work around this to support that in the future.